.github/workflows/ci.yaml

```diff
@@ -30,8 +30,16 @@ jobs:
         with:
           version: 'v0.12.0'
 
+      - name: Install Noto Serif CJK fonts
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y fonts-noto-cjk
+          fc-cache -f -v
+
```

Comment:
I tried to first install Noto fonts in the GitHub action before running the tests, but had no luck. I suspect it's because different systems render CJK characters slightly differently (though almost indistinguishable), as illustrated below. This is a test results where typst-test will flag it as failed.

![1737246437398](https://jtchen.s3.ap-northeast-1.amazonaws.com/v1/img/2025/01/18/1737246437398.png)

As the goal for this package is counting words, it might be acceptable to just make CJK characters rendered as block to ensure consistency across machines. You can test CJK font rendering using this font family sequence if you'd like:

```typst
#set text(font: ("Noto Serif CJK SC", "Noto Serif CJK HK", "Noto Serif CJK JP", "Noto Serif CJK KR"), fallback: false)
```